tutorials/how_to/run-benchmarks.md

```diff
@@ -0,0 +1,379 @@
+- Machine Type: Specify the machine type and configuration (e.g., `c2-standard-4`, `c3-standard-44`) using `resources.MachineGroup`.
+
+**3- Run the Benchmark** 
+
+Use `.run(num_repeats=2)` to execute all added runs, repeating each one twice. 
+*Repeating runs helps ensure consistent and reliable performance metrics.*
+
+The following code example demonstrates the setup for testing `c2-standard` and 
+`c3-standard` machines:
+
+```python
+from inductiva import benchmarks, simulators, resources
+
+benchmarks.Benchmark(name="splishsplash-fluid-cube") \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c2-standard-4")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",             
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c2-standard-8")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",             
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c2-standard-16")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c2-standard-30")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c2-standard-60")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c3-standard-4")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c3-standard-8")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c3-standard-22")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c3-standard-44")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c3-standard-88")) \
+    .add_run(simulator=simulators.SplishSplash(),
+             input_dir="splishsplash-base-dir",
+             sim_config_filename="config.json",
+             on=resources.MachineGroup("c3-standard-176")) \
+    .run(num_repeats=2)
+```
+
+## Step 3: Refine the Code to Improve Data Readability
```

Comment:
Should we refer to this as a step? Maybe it should be part of step 2

Reply:
i would say let us just add one step just for refining the code, and merge step 3 with 4 so that step 2 doesnt have so many things going for it.